### PR TITLE
PXB-1842: v2.4.14 - release notes

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.14.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+New Features
+================================================================================
+
+- Amazon S3 is now supported in |xbcloud|. More information in
+  :jirabug:`PXB-1813`.
+
+Bugs Fixed
+================================================================================
+
+- When the row format was changed, the server could crash during the incremental
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.14.rst]: when row format was change *during the backup*, *xtrabackup* could crash during incremental prepare stage

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.14.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  backup at the prepare stage. Bug fixed :jirabug:`PXB-1824`.
+- If compressed InnoDB undo tablespaces were not removed beforehand, the
+  incremental backup could crash at the prepare stage. Bug fixed
+  :jirabug:`PXB-1552`.
+- In some cases, |xbcloud| did not exit when a piped command failed. Bug fixed
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.14.rst]: this bug in not present in any released version, no need to mention in release notes

**Review by gl-sergei — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.14.rst (hunk 2 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.14.rst]: Ready for review

**Review by gl-sergei — APPROVED** [review]

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.14.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+   $ xtrabackup --backup --stream=xbstream --extra-lsndir=/tmp --target-dir=/tmp | \
+   xbcloud put --storage=s3 \
+   --s3-endpoint='s3.amazonaws.com' \
+   --s3-access-key='YOUR-ACCESSKEYID' \
+   --s3-secret-key='YOUR-SECRETACCESSKEY' \
+   --s3-bucket='mysql_backups'
+   --parallel=10 \
+   ${date}-full_backup
+
```
Comment (hrvojem) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.14.rst:24]: this should be part of xbcloud [docs](https://www.percona.com/doc/percona-xtrabackup/2.4/xbcloud/xbcloud.html)

**Review by hrvojem — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.14.rst (hunk 3 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.14.rst:24]: I know. the related pull request will be added shortly based on PXB-1813

**Review by hrvojem — APPROVED** [review]

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.14.rst (hunk 3 above)
Comment (hrvojem) [reply storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.14.rst:24]: ok

**Pull request merged**